tests/foreman/installer/test_infoblox.py

```diff
@@ -16,6 +16,117 @@
 """
 import pytest
 
+from robottelo.helpers import InstallerCommand
+
+
+def register_satellite(sat):
+    sat_version = str(sat.execute('rpm -q satellite | cut -d- -f2 | cut -d. -f1,2 | tr -d "\n"'))
```

Comment:
I think this is not required, the satellite version dependent configuration will be updated based on the selected satellite_version in the robottelo conf file.